test/sequential/sequencerStaking.cancel.test.ts

```diff
@@ -0,0 +1,403 @@
+/*
+ *
+ * @group sequencerStaking
+ */
+
+import {
+  ChainName,
+  SequencerStaking,
+} from "../../utils/rollDown/SequencerStaking";
+import { L2Update, Rolldown } from "../../utils/rollDown/Rolldown";
+import { BN_MILLION, MangataGenericEvent, signTx } from "gasp-sdk";
+import { getApi, initApi } from "../../utils/api";
+import { setupApi, setupUsers } from "../../utils/setup";
+import { expectExtrinsicSucceed, waitForNBlocks } from "../../utils/utils";
+import { AssetWallet, User } from "../../utils/User";
+import { Sudo } from "../../utils/sudo";
+import {
+  waitNewBlock,
+  waitSudoOperationSuccess,
+} from "../../utils/eventListeners";
+import { Assets } from "../../utils/Assets";
+import { BN_ZERO } from "@polkadot/util";
+import { MGA_ASSET_ID } from "../../utils/Constants";
+
+async function waitSequencerAsSelected(seq: User, n: number = 80) {
+  let selectedSequencer: any;
+  const api = getApi();
+  selectedSequencer = (
+    await api.query.sequencerStaking.selectedSequencer()
+  ).toHuman();
+  while (selectedSequencer.Ethereum !== seq.keyRingPair.address && n > 0) {
+    await waitNewBlock();
+    selectedSequencer = (
+      await api.query.sequencerStaking.selectedSequencer()
+    ).toHuman();
+    n--;
+  }
+}
+
+async function createACollatorUser() {
+  const [user] = setupUsers();
+  await Sudo.asSudoFinalized(Assets.mintNative(user));
+  return user;
+}
+
+async function leaveSequencingIfAlreadySequencer(userAddr: string) {
```

Comment:
is it duplicated from some other test? 
Plz refactor it to an external class and reuse it in both tests

Reply:
It was moved to Rolldown.ts